tieredhashing/tiered_hashing.go

```diff
@@ -241,31 +253,43 @@ func (t *TieredHashing) GetPerf() map[string]*NodePerf {
 	return t.nodes
 }
 
-func (t *TieredHashing) AddOrchestratorNodes(nodes []string) (added, alreadyRemoved, removedAndAddedBack int) {
+func (t *TieredHashing) GetSentinelCid(ip string) (string, error) {
+	if node, ok := t.nodes[ip]; ok {
+		return node.SentinelCid, nil
```

Comment:
should we delete the sentinel once we get it, rather than potentially making the same sentinel request multiple times?

Reply:
I don't see a significant risk of requesting the same sentinel cid multiple times. Although, I think this is dependent on the frequency that we update the sentinel cids. Currently, it is hourly so I think it is enough time to rotate the cids without making too many duplicate requests.

Reply:
One issue I think exists with the current implementation though is that we are only making sentinel cid requests to the top performing nodes in the caboose pool. This is because the sentinel cid request code is triggered when a saturn is picked to fetch a cid. 

So maybe an improvement over this is to have the sentinel cid code instead randomly choose a node from the pool and then delete its sentinel cid? That way we make sure we're making our sentinel cid requests uniformly across all nodes. Thoughts?